packages/paypal-express-integration/src/create-paypal-express-button-strategy.ts

```diff
@@ -0,0 +1,24 @@
+import { createFormPoster } from '@bigcommerce/form-poster';
+import { getScriptLoader } from '@bigcommerce/script-loader';
+
+import {
+    CheckoutButtonStrategyFactory,
+    toResolvableModule,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import { PaypalExpressButtonStrategy, PaypalExpressScriptLoader } from './index';
+
+const createPaypalExpressButtonStrategy: CheckoutButtonStrategyFactory<
+    PaypalExpressButtonStrategy
+> = (paymentIntegrationService) => {
+    const { getHost } = paymentIntegrationService.getState();
+
+    return new PaypalExpressButtonStrategy(
+        paymentIntegrationService,
+        new PaypalExpressScriptLoader(getScriptLoader()),
+        createFormPoster(),
+        getHost(),
```

Comment:
I think, `getHost` can be removed, because PaypalExpressButtonStrategy does not use it in the constructor

Reply:
removed